submitr/rclone/rclone_google.py

```diff
@@ -0,0 +1,216 @@
+from __future__ import annotations
+import io
+import json
+import os
+import requests
+from typing import Callable, Optional, Union
+from dcicutils.file_utils import normalize_path
+from dcicutils.misc_utils import create_dict, normalize_string, PRINT
+from submitr.rclone.rclone_commands import RCloneCommands
+from submitr.rclone.rclone_config import RCloneConfig, RCloneCredentials
+from submitr.rclone.rclone_installation import RCloneInstallation
+from submitr.rclone.rclone_utils import cloud_path
+from submitr.utils import chars
+
+
+class RCloneGoogle(RCloneConfig):
```

Comment:
Yeah I originally had it named that, and changed because it's not *really* just a config - it has calls to do things - e.g. to get the file size and checksum - So I feel the abstract base class is actually misnamed and it should maybe be RCloneTarget  and these should be either like RCloneAmazon or RCloneTargetAmazon..